src/core/maps/const_occupancy_estimator.h

```diff
@@ -3,10 +3,24 @@
 
 #include "cell_occupancy_estimator.h"
 
+/**
+ * \brief ConstEstimator is responsible for occupancy estimation strategy that returns probability of being occupied for a cell depending only on the cell status predicted by a sensor.
+ */
 class ConstOccupancyEstimator : public CellOccupancyEstimator {
 public:
+/**
+ * This constructor uses constructor of CellOccupancyEstimator
+ * \param occ probability of being a cell occupied
+ * \param empty probability that cell is empty
+ */
   ConstOccupancyEstimator(double occ, double empty) :
     CellOccupancyEstimator(occ, empty) {}
+/**
+ * Returns base probability that cell is occupied uses only the cell status returned by a sensor for estimator.
+ * \param beam Laser beam (ignored)
+ * \param cell_bnds geometric representation of the cell (ignored)
+ * \param is_occ flag is responsible for information about cell occupancy
```

Comment:
"<...> is_occ flag that indicates whether the cell is occupied according to sensor data"